mne/io/eeglab/eeglab.py

```diff
@@ -445,3 +445,46 @@ def __init__(self, input_fname, events=None, event_id=None, tmin=0,
             reject=reject, flat=flat, reject_tmin=reject_tmin,
             reject_tmax=reject_tmax, add_eeg_ref=False, verbose=verbose)
         logger.info('Ready.')
+
+
+def _create_events_from_eeglab_raw(fname, event_id=dict()):
```

Comment:
You should add a stim channel to the data before you call the `_BaseEpochs` and also update the info dict accordingly. Don't return anything from the function.